Break down Sources into individual sections in Docs

iamernie opened the pull request:
The start of breaking down sources into individual sections that include more detailed instructions and pictures on how to set them up and configure them.

iamernie committed:
Create index.mdx

Add Wekan example

iamernie committed:
updated to include wekan entry

iamernie committed:
Merge branch 'goauthentik:master' into master

iamernie committed:
Merge branch 'goauthentik:master' into master

iamernie committed:
Update and rename website/docs/sources.md to website/docs/sources/index.md

Break Sources into individual pages.

iamernie committed:
Update and rename website/docs/sources/index.md to website/docs/sources/ldap/index.md

iamernie committed:
Create index.md

iamernie committed:
Update index.md

iamernie committed:
Update index.md

iamernie committed:
Create index.md

iamernie committed:
Create index.md

iamernie committed:
Create index.md

iamernie committed:
Update index.md

iamernie committed:
Update index.md

iamernie committed:
Update index.md

iamernie committed:
Merge branch 'goauthentik:master' into master

iamernie committed:
Create index.md

iamernie committed:
discord images

iamernie committed:
spacing

iamernie committed:
Added discord

iamernie committed:
discord changes

iamernie committed:
Added sources breakdown to the sidebar

iamernie committed:
Fixed the saml title

iamernie committed:
Added github examples

iamernie committed:
fixed formatting

BeryJu commented:
@iamernie whats the status, can I merge this, or are you still making changes? 

I do have a few small change requests, can you move the files from /docs/sources to /docs/integrations/sources?

Also there are some errors in the sidebar, see https://app.netlify.com/sites/authentik/deploys/60cf8253e1343a0008a40fe1

iamernie committed:
Changed file path, updated sidebar, added google.

iamernie committed:
Merge branch 'goauthentik:master' into master

iamernie committed:
fixed a spelling mistake

iamernie committed:
Merge branch 'master' of github.com:iamernie/authentik

BeryJu commented on website/docs/integrations/sources/github/index.md:
```diff
@@ -0,0 +1,58 @@
+
+![Register OAuth App](githubdeveloper1.png)
+
+2. **Application Name:** <Choose a name users will recognize ie: Authentik>
```
Since HTML is allowed in markdown, < and > are reserved characters. I think in this case the chars can just be removed, and for the other cases, where another field is referenced, you can do 
```
`<foo>`
```

iamernie replied:
Sounds Good.  I'll update the pages now.

iamernie replied:
Formatting has been cleaned up

iamernie commented:
Hi @BeryJu I've moved all the files to the new location.  I also relocated the "Sources" page and moved it into the Integrations/Sources too.  Sidebar.js is fixed and this should be ready to merge.

iamernie committed:
Cleaned up formatting

iamernie committed:
Fixed Notes
